templates/server/src/main/resources/archetype-resources/core/src/main/java/__packageInPathFormat__/general/common/impl/config/ApplicationObjectMapperFactory.java

```diff
@@ -32,6 +34,9 @@ public ApplicationObjectMapperFactory() {
     // register spring-data Pageable
     module.addSerializer(Pageable.class, new PageableJsonSerializer());
     module.addDeserializer(Pageable.class, new PageableJsonDeserializer());
+    // register spring-data Page
+    module.addAbstractTypeMapping(Page.class, JsonPage.class);
+    module.setMixInAnnotation(Page.class, JsonPage.class);
```

Comment:
fine, fixes our problem.
As I already stated we should IMHO provide the fundamental config in `ObjectMapperFactory` in `devon4j` module in a protected method. We have done things wrong here several times and bringing out bugfixes to the archetype does not help the projects that have already started from the archetype.
IMHO all what is configured here could be done in a new projected method called e.g. `ObjectMapperFactory.initMapping()`.

Reply:
Done